distribution/tools/launchers/src/main/java/org/elasticsearch/tools/launchers/JvmErgonomics.java

```diff
@@ -56,11 +56,11 @@ private JvmErgonomics() {
         final Map<String, Optional<String>> finalJvmOptions = finalJvmOptions(userDefinedJvmOptions);
         final long heapSize = extractHeapSize(finalJvmOptions);
         final Map<String, String> systemProperties = extractSystemProperties(userDefinedJvmOptions);
-        if (systemProperties.containsKey("io.netty.allocator.type") == false) {
+        if (systemProperties.containsKey("es.unsafe.use_unpooled_allocator") == false) {
```

Comment:
this setting does not need the unsafe prefix, that's only relevant for `es.unsafe.use_netty_default_allocator`